arangodb-foxx-services/tests/aql/example.spec.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import myCoolFunction = require('../../aql/example.func')
+import myCoolFunction = require('../../src/aql/example.func')
```

Comment:
@wajda is this test example really needed?

Reply:
well, I would say yes. There is currently no user defined AQL function used in Spline, but there used to be, and could be in the future. And since AQL UDFs are deployed using a separate procedure, we have created a set of scripts and configuration to make this process as simple as merely dropping a file in a source folder. There is however a couple of conventions that must be followed in order for the UDF and the build process to work correctly, so a clear example has to be placed somewhere. And this is the place. We could simply remove this UDF build part along with the UDF source folder due to redundancy, and we'll probably do it later at some point. But for now I'd like to keep it around and ready just in case we decide to create another UDF in the project.